tests_e2e/orchestrator/lib/agent_test_suite.py

```diff
@@ -148,17 +151,12 @@ def _set_context(self, node: Node, variables: Dict[str, Any], lisa_log_path: str
         self.__context.log_path = self._get_log_path(variables, lisa_log_path)
         self.__context.log = log
         self.__context.node = node
-        self.__context.is_vhd = self._get_required_parameter(variables, "c_vhd") != ""
-        self.__context.image_name = f"{node.os.name}-vhd" if self.__context.is_vhd else self._get_required_parameter(variables, "c_name")
+        self.__context.is_vhd = self._get_optional_parameter(variables, "c_vhd") != ""
+        self.__context.image_name = f"{node.os.name}-vhd" if self.__context.is_vhd else self._get_required_parameter(variables, "c_env_name")
```

Comment:
I added a "vm_name" parameter for existing VMs, so I renamed "c_name" to "c_env_name"